src/commands/sgd/source/delta.ts

```diff
@@ -84,6 +92,8 @@ export default class SourceDeltaGenerate extends SfdxCommand {
         repo: this.flags.repo,
         ignoreWhitespace: this.flags['ignore-whitespace'],
         generateDelta: this.flags['generate-delta'],
+        include: this.flags['include'],
```

Comment:
**suggestion(if-minor):** could use the attributes 

```suggestion
        include: this.flags.include,
```

Reply:
Yeah, I'm surprised that `prettier`, or `eslint` didn't catch this.